src/App.js

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
-import './App.css';
 import SigninForm from './components/signinForm';
 import SignupForm from './components/signupForm';
+import { BrowserRouter as Router,Switch, Route } from 'react-router-dom';
 
 function App() {
   return (
-    <div>
-    <SigninForm/>
-  
-    </div>
+    <Router>
+      <Switch>
+        <Route exact path="/"
```

Comment:
I think it's better to have signin as default, then a button leading to signup in case the user doesn't yet have an account.